<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelNombreTienda">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDirWeb">
         <property name="text">
          <string>Dirección Web:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDirFisica">
         <property name="text">
          <string>Dirección Física:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTelefono">
         <property name="text">
          <string>Teléfono:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnSiguientePagina">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>340</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Producto</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelTituloTienda">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNombre"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDirWeb"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDirFisica"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTelefono"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>160</width>
        <height>116</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnCrearTienda">
         <property name="text">
          <string>Crear Tienda</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCargarTienda">
         <property name="text">
          <string>Cargar Tienda</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGuardarTienda">
         <property name="text">
          <string>Guardar Tienda</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLimpiar">
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>281</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>161</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="btnAgregar">
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModificar">
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminar">
         <property name="text">
          <string>Eliminar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnRegresarTienda">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Tienda</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
